Sources/Internal/FrameProvider.swift

```diff
@@ -273,6 +295,7 @@ final class FrameProvider {
         daySize.height - content.verticalDayMargin -
         (maxNumberOfWeekRowsPerMonth * daySize.height) -
         ((maxNumberOfWeekRowsPerMonth - 1) * content.verticalDayMargin) -
+        monthFooterHeight -
         content.monthDayInsets.bottom
```

Comment:
Can we swap the order of this subtraction so that it reflects the actual ordering of the values? Basically, the `monthDayInsets.bottom` should be between the days and the month footer, not below the month footer. Like this:

![image](https://user-images.githubusercontent.com/746571/100386241-e9f45280-2fc8-11eb-9551-ac40b60dc835.png)

Reply:
Updated as per comments